pkg/events/parsers/data_parsers.go

```diff
@@ -13,11 +13,27 @@ import (
 	"github.com/aquasecurity/tracee/pkg/utils/environment"
 )
 
-type SystemFunctionArgument interface {
+
+type systemFunctionArgument interface {
```

Comment:
Why making the interface private?
Isn't the idea behind interfaces to be exposed between different packages?

Reply:
It is not used (imported) so I just made it private via https://github.com/aquasecurity/tracee/commit/c07114173ce275f33a73e3bdf468d59118303e2e by creating a public type with same name - since the type or its pointer is cheaper than dealing with an interface.

Reply:
As we discussed I'm going to remove the private interface (enforcer). 👍🏼

Reply:
Well, in the future we'll remove that interface, other still not refactored parsers consumes it.